inc/Post_Thumbnails/Component.php

```diff
@@ -40,4 +42,12 @@ public function initialize() {
 	public function action_add_post_thumbnail_support() {
 		add_theme_support( 'post-thumbnails' );
 	}
+
+	/**
+	 * Adds custom image sizes.
+	 */
+	public function action_add_image_sizes() {
+		add_image_size( 'wp-rig-featured', 720, 480, true );
+		add_image_size( 'wp-rig-thumbnail', 720, 308, true );
```

Comment:
@benoitchantre We don't need our own thumbnail size. Where `wp-rig-thumbnail` is used, we should be using just `post-thumbnail`.

Reply:
@benoitchantre I will take care of this so we can merge for the release.